Fill missing sections in README (#625)
google/jsonnet: README.md

### Running tests

```

> make && make libjsonnet.so &&  ./tests.sh

Reviewer: You can just do `make test` to get this (and it also builds the little C programs that are tested).

```

### Locally serving the website

> First, you need to build JS version of Jsonnet (requires emscripten):

Reviewer 2: ```suggestion
First, you need to build the JavaScript version of Jsonnet (which requires [_emscripten_](https://emscripten.org/)):
```

```
make doc/js/libjsonnet.js
```

> Then inside `doc` directory you can generate and serve the website using [Jekyll](https://jekyllrb.com/):

Reviewer 2: ```suggestion
Then, inside the _doc_ directory, you can generate and serve the website using [Jekyll](https://jekyllrb.com/):
```

> Then, inside the _doc_ directory, you can generate and serve the website using [Jekyll](https://jekyllrb.com/):

Reviewer: There is a script `tools/scripts/serve_docs.sh` which does this but without --livereload.  Perhaps it should use --livereload though.  It must be run from the root of the repo (but it has friendly checks for that).

```
jekyll serve --watch --livereload
```
Or you can do the same more concisely:
```
jekyll s -w -l
```

> This should serve the website, automatically rebuild when you change underlying files and automatically

Reviewer 2: ```suggestion
This should serve the website, automatically rebuild when you change any underlying files, and automatically
```
